src/smk/tool/geomark/tool-geomark.js

```diff
@@ -243,23 +274,24 @@ include.module( 'tool-geomark', [
                     url: geomarkUrl + '/feature.geojson',
                     dataType: 'json',
                     traditional: true,
-                    success: function(geomarkFeature) {
-                        var geometryLayer = L.geoJSON(geomarkFeature.geometry, {
+                    success: function(geomarkGeoJson) {
+                        const geometry = self.getGeometry(geomarkGeoJson);
+                        const geometryLayer = L.geoJSON(geometry, {
```

Comment:
An L.geoJson layer can accept an input parameter of geojson, so there is no need to parse out a geometry here, you can just do
```
const geojsonLayer = L.geoJson(geomarkGeoJson, {
  style: function(feature) {
    return {
      color: CUSTOM_COLOR
    };
  }
);
```

Reply:
Thanks! I've confirmed we can successfully make layers from the received GeoJSON (polygons and multipolygons) using prod, delivery, and test Geomark environments and their varied JSON structures.

I'm happy to rely on Leaflet's parsing and not have to maintain our own :)